lib/oli_web/components/delivery/content/content_table_model.ex

```diff
@@ -0,0 +1,112 @@
+defmodule OliWeb.Components.Delivery.ContentTableModel do
+  use Phoenix.Component
+  alias OliWeb.Common.Table.{ColumnSpec, SortableTableModel}
+  alias OliWeb.Router.Helpers, as: Routes
+
+  def new(containers, container_column_name, section_slug) do
+    column_specs = [
+      %ColumnSpec{
+        name: :container_name,
+        label: container_column_name,
+        render_fn: &__MODULE__.render_name_column/3,
+        th_class: "pl-10 instructor_dashboard_th"
+      },
+      %ColumnSpec{
+        name: :student_completion,
+        label: "STUDENT COMPLETION",
+        render_fn: &__MODULE__.render_student_completion/3,
+        th_class: "instructor_dashboard_th"
+      },
+      %ColumnSpec{
+        name: :student_mastery,
+        label: "STUDENT MASTERY",
+        render_fn: &__MODULE__.stub_student_mastery/3,
+        th_class: "instructor_dashboard_th"
+      },
+      %ColumnSpec{
+        name: :student_engagement,
+        label: "STUDENT ENGAGEMENT",
+        render_fn: &__MODULE__.stub_student_engagement/3,
+        th_class: "instructor_dashboard_th"
+      }
+    ]
+
+    SortableTableModel.new(
+      rows: containers,
+      column_specs: column_specs,
+      event_suffix: "",
+      id_field: [:id],
+      data: %{section_slug: section_slug}
+    )
+  end
+
+  def render_name_column(
+        assigns,
+        container,
+        column_spec
+      ) do
+    url_params =
+      case column_spec.label do
+        "PAGES" -> %{page_id: container.id}
+        _ -> %{container_id: container.id}
+      end
+
+    assigns =
+      Map.merge(assigns, %{
+        progress: parse_progress(container.progress),
+        title: container.title,
+        url_params: url_params,
+        section_slug: assigns.model.data.section_slug
```

Comment:
I'm not entirely sure, but I think you could call 'assigns.section_slug' directly here, since you already defined section_slug in the data attribute of the SortableTableModel definition above.